packages/mantine/src/components/table/table-actions/TableActions.tsx

```diff
@@ -1,13 +1,52 @@
-import {Grid, Group} from '@mantine/core';
-import {ReactElement, ReactNode} from 'react';
+import {Factory, Grid, GridColProps, Group, useProps} from '@mantine/core';
+import {ForwardedRef, ReactElement, ReactNode} from 'react';
 
+import {CustomComponentThemeExtend, identity} from '../../../utils';
 import {TableComponentsOrder} from '../Table';
-import {useTable} from '../TableContext';
-import TableActionsClasses from './TableActions.module.css';
-import {TableActionsProps} from './TableActions.types';
+import {useTable, useTableStyles} from '../TableContext';
 
-export const TableActions = <T,>({children, ...others}: TableActionsProps<T>): ReactElement => {
+export type TableActionsStylesNames = 'actionsRoot' | 'actionsGroup';
+
+export interface TableActionsProps<T> extends Omit<GridColProps, 'children'> {
+    /**
+     * Function that return components for the selected row or selected rows when multi row selection is enabled
+     *
+     * @param datum the data of the selected row(s)
+     * @example
+     * <Table.Actions<MyType>>
+     *     {(datum: MyType) => (
+     *         <Button
+     *             component={Link}
+     *             to={`edit/${datum.id}`}
+     *             leftIcon={<EditSize24Px />}
+     *             variant="subtle"
+     *             color="navyBlue.8"
```

Comment:
We could give an example that actually exist

```suggestion
     *             color="navy.8"
```